api/environments/job_handler.go

```diff
@@ -165,6 +166,36 @@ func (eh EnvironmentHandler) StopAllBatches(ctx context.Context, appName, envNam
 	return jobSchedulerBatch.StopAllRadixBatches(ctx, eh.accounts.UserAccount.RadixClient, appName, envName, jobComponentName, kube.RadixBatchTypeBatch)
 }
 
+// StopAllBatchesAndJobsForJobComponent Stop all scheduled batches in the job-component
+func (eh EnvironmentHandler) StopAllBatchesAndJobsForJobComponent(ctx context.Context, appName, envName, jobComponentName string) error {
+	errs := slice.Map([]kube.RadixBatchType{kube.RadixBatchTypeBatch, kube.RadixBatchTypeJob}, func(batchType kube.RadixBatchType) error {
+		return jobSchedulerBatch.StopAllRadixBatches(ctx, eh.accounts.UserAccount.RadixClient, appName, envName, jobComponentName, batchType)
+	})
```

Comment:
Could we simplify this logic? and match the style of `StopAllBatchesAndJobsForEnvironment`?

```suggestion
	var errs []error

	if err := jobSchedulerBatch.StopAllRadixBatches(ctx, eh.accounts.UserAccount.RadixClient, appName, envName, jobComponentName, kube.RadixBatchTypeBatch); err != nil {
		errs = append(errs, err)
	}
	if err := jobSchedulerBatch.StopAllRadixBatches(ctx, eh.accounts.UserAccount.RadixClient, appName, envName, jobComponentName, kube.RadixBatchTypeJob); err != nil {
		errs = append(errs, err)
	}
	
```